src/components/v-button/v-button.vue

```diff
@@ -61,17 +62,36 @@ export default defineComponent({
 			type: Boolean,
 			default: false
 		},
+		value: {
+			type: String,
+			default: null
+		},
 		...sizeProps
 	},
 	setup(props, { emit }) {
 		const sizeClass = useSizeClass(props);
 
 		const component = computed<string>(() => (props.to ? 'router-link' : 'button'));
 
-		return { sizeClass, onClick, component };
+		let active;
+		let toggle = () => {return}
+
+
+		try {
+			const groupable = useGroupable(props.value) || {active: undefined, toggle: undefined};
+			active = groupable.active
+			toggle = groupable.toggle
+		} catch (error) {
+			active = ref(false)
+			toggle = () => {return}
+			
+		}
+
+		return { sizeClass, onClick, component, active, toggle };
 
 		function onClick(event: MouseEvent) {
 			if (props.loading === true) return;
+			toggle()
```

Comment:
Lets add a little comment in here indicating what this does

```suggestion
			// Toggles the active state in the parent groupable element. Allows buttons to work ootb in button-groups
			toggle();
```